src/_scss/pages/search/searchSidebar.scss

```diff
@@ -3,16 +3,62 @@
     @import '../../elements/_inputs';
     @import '../../elements/_labels';
 
+    .filter-header {
+        font-size: $base-font-size;
+        font-weight: bold;
+        padding-bottom: 1rem;
+        padding-left: 1rem;
+        display: inline-block;
+    }
+
 	.search-option {
 		margin-bottom:1rem;
 
         label {
             @include usa-label;
         }
-	}
 
-	.search-option b::after {
-		content: ' \02193';
+        svg {
+            fill: $color-gray;
+            height: 2rem;
+            width: 2rem;
+            position:relative;
+            cursor: pointer;
+        }
+
+        .toggle {
```

Comment:
Don't just copy-paste the Sass, rename and/or otherwise constrain the style so it won't conflict or be affected by changes to the source it was copied from  (the award subtype button).